src/main/java/net/bull/javamelody/LiferayMonitoringFilter.java

```diff
@@ -30,9 +30,9 @@
 
 import com.liferay.portal.kernel.exception.PortalException;
 import com.liferay.portal.kernel.exception.SystemException;
-import com.liferay.portal.model.RoleConstants;
-import com.liferay.portal.service.UserLocalServiceUtil;
-import com.liferay.portal.util.PortalUtil;
+import com.liferay.portal.kernel.model.RoleConstants;
+import com.liferay.portal.kernel.service.UserLocalServiceUtil;
+import com.liferay.portal.kernel.util.PortalUtil;
```

Comment:
The hook needs to be backward compatible with Liferay 6.1+.
So the previous dependency `portal-service` should be added back and the code should use one or the other classes are available at runtime.